app/views/providers/about_the_financial_assessments/show.html.erb

```diff
@@ -1,22 +1,37 @@
 <%= page_template(page_title: t('.title')) do %></h1>
   <p class="govuk-body"><%= t '.sign_in_para' %></p>
 
-  <p class="govuk-body"><%= t '.share_bank_accounts_para' %></p>
-
-  <h2 class="govuk-heading-m"><%= t '.list.title' %></h2>
-
   <p class="govuk-body">
     <%= list_from_translation_path 'providers.about_the_financial_assessments.show.list' %>
   </p>
 
+  <h2 class="govuk-heading-m"><%= t '.section_1.title' %></h2>
+
+  <p class="govuk-body"><%= t '.section_1.detail_use' %></p>
+
+  <p class="govuk-body"><%= t '.section_1.sign_in_save' %></p>
+
   <h2 class="govuk-heading-m">
-    <%= t '.section_1.heading' %>
+    <%= t '.section_2.heading' %>
   </h2>
 
   <p>
-    <%= t '.section_1.text' %>
+    <%= t '.section_2.text' %>
   </p>
 
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__value">
+          <%= @applicant.email %>
+        </dt>
+        <dd class="govuk-summary-list__actions">
+          <%= link_to(providers_legal_aid_application_email_address_path(anchor: :email), class: 'govuk-link change-link') do %>
```

Comment:
Would this work?
```ruby
<%= link_to(t('generic.change'), providers_legal_aid_application_email_address_path(anchor: :email), class: 'govuk-link change-link') %>
```

With `link_to` you can pass the `text` as the first parameter https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to
You don't need a block